pkg/api/interceptor.go

```diff
@@ -152,7 +153,7 @@ func (wa *WalletAuthorizer) authorize(ctx context.Context, req interface{}, wall
 	return nil
 }
 
-func (wa *WalletAuthorizer) applyLimits(ctx context.Context, req interface{}, wallet types.WalletAddr) error {
+func (wa *WalletAuthorizer) applyLimits(ctx context.Context, fullMethod string, req interface{}, wallet types.WalletAddr) error {
```

Comment:
I ended up not using the `fullMethod` yet, but if we decide to introduce limits for different types of stream requests, we will need it because we don't get the unmarshaled request in that case.